src/Aquifer.Common/Clients/SendGridClient.cs

```diff
@@ -0,0 +1,49 @@
+using Aquifer.Data;
+using Microsoft.Extensions.Configuration;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+
+namespace Aquifer.Common.Clients;
+
+public interface ISendGridClient
+{
+    Task<Response> SendEmail(AquiferDbContext dbContext, SendGridEmailConfiguration emailConfiguration, CancellationToken ct);
```

Comment:
I'd inject the AquiferDbContext in the constructor.

Reply:
though I'm not even seeing where it's being used

Reply:
Thanks for catching that. I don't use it. I'll take that out.